### DISCUSSION TITLE
How to disable an entity's fall damage?

### DISCUSSION THREAD

**Discussion by GitUser14975** [FabricMC/fabric-api#3176] [Mod Dev Support] [answered]
### Description

I created an entity that flies, but sometimes when it hits the ground it gets damaged. How can I disable that? The fall function seems to be private and I'm unable to override it

**Comment by haykam821** [comment, marked as answer]
Entities in the `minecraft:fall_damage_immune` entity type tag do not take fall damage. You can [add your entity type to the tag](https://fabricmc.net/wiki/tutorial:tags) to achieve your desired effect.
